cerberus-audit-logger-athena/src/main/java/ch/qos/logback/core/rolling/AuditLogsS3TimeBasedRollingPolicy.java

```diff
@@ -65,4 +66,9 @@ public void rollover() throws RolloverFailure {
       }
     }
   }
+
+  void superRollOver() {
```

Comment:
What is this for?

Reply:
this method is to mock super class rollover method